parachain/pallets/control/src/tests.rs

```diff
@@ -501,11 +502,13 @@ fn check_sibling_sovereign_account() {
 	new_test_ext().execute_with(|| {
 		let para_id = 1001;
 		let sovereign_account = sibling_sovereign_account::<Test>(para_id.into());
+		let sovereign_account_raw = sibling_sovereign_account_raw(para_id.into());
```

Comment:
Obsolete print statements?

Reply:
These prints statements are for showing us addresses. Dont think the are causing any harm here.